numpy/__init__.py

```diff
@@ -382,12 +382,13 @@ def _mac_os_check():
             if len(w) > 0:
                 error_message = "{}: {}".format(w[-1].category.__name__, str(w[-1].message))
                 msg = (
-                    "Polyfit sanity test emitted a warning, most likely due "
-                    "to using a buggy Accelerate backend. If you compiled "
-                    "yourself, more information is available at "
-                    "https://numpy.org/doc/stable/user/building.html#accelerated-blas-lapack-libraries "
-                    "Otherwise report this to the vendor "
-                    "that provided NumPy.\n{}\n".format(error_message))
+                    "\nPolyfit sanity test emitted a warning, most likely due\n"
+                    "to using a buggy Accelerate backend. If you compiled\n"
+                    "yourself, more information is available at\n"
+                    "https://numpy.org/doc/stable/user/building.html#accelerated-blas-lapack-libraries\n"
+                    "Otherwise report this to the vendor\n"
+                    "that provided NumPy.\n{}\n".format(error_message)
```

Comment:
I think if you're going to add manual line-breaks, you should do it in a way that's semantically meaningful, as a terminal can wrap the message to columns just fine. Perhaps:
```
Polyfit sanity test emitted a warning, most likely due to using a buggy Accelerate backend.
If you compiled yourself, more information is available at:
  https://numpy.org/doc/stable/user/building.html#accelerated-blas-lapack-libraries"
Otherwise report this to the vendor that provided NumPy.
```

Reply:
I think that would be fine. I believe the motivating use-case was that someone ran into this in some sort of environment (IDE or maybe a notebook) where the warning was placed in an element with horizontal scrolling (similar to the second suggestion above).

I added line breaks to match the linebreaks in the source as I figured that was the most likely to avoid scrolling elements, but your first suggestion looks like it will fit the bill as well :+1: